LuaUI/Widgets/test1_sg.lua

```diff
@@ -0,0 +1,14 @@
+VFS.Include("LuaRules/Utilities/GetSpiralGenerator.lua")
+
+local spiralWithExtraParameters = Spring.Utilities.GetSpiralGenerator(12, 34, 3, "n", false)
```

Comment:
The test specified in #5197 looks different.
```suggestion
local spiralWithExtraParameters = Spring.Utilities.GetSpiralGenerator(12, 34, { step = 3, startingDirection = "n",  clockwise = false })
```

Reply:
In my defense, I thought these (aka { and }) was causing a syntax error when i tried to put it into the start_unit_setup gadget before I reverted it back to normal and before I found out that test widget weren't even properly loaded. I will just revert it back and then test it, give me a moment.

Reply:
Perhaps look around other Lua widgets and check how various functions manage to pass and accept { tables }.